test-data-prepare/root-org/root-org.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Rename test-data-perpare to test-data-preparation
Rename root-org.sh to create-root-org.sh